priv/repo/migrations/20240729191431_create_badges.exs

```diff
@@ -10,7 +10,9 @@ defmodule Safira.Repo.Migrations.CreateBadges do
       add :begin, :utc_datetime
       add :end, :utc_datetime
       add :tokens, :integer
+      add :entries, :integer
       add :counts_for_day, :boolean
+      add :givable, :boolean
       add :category_id, references(:badge_categories, on_delete: :delete_all, type: :binary_id)
```

Comment:
I don't think it is a good idea to be changing a migration file with the app is prod.